examples/ct/pem_cert_pool.go

```diff
@@ -67,6 +69,19 @@ func (p *PEMCertPool) AppendCertsFromPEM(pemCerts []byte) (ok bool) {
 	return
 }
 
+// AppendCertsFromPEMFile adds certs from a file that contains concatenated PEM data.
+func (p *PEMCertPool) AppendCertsFromPEMFile(pemFile string) error {
+	pemData, err := ioutil.ReadFile(pemFile)
+	if err != nil {
+		return fmt.Errorf("failed to load PEM certs file: %v", err)
+	}
+
+	if !p.AppendCertsFromPEM(pemData) {
```

Comment:
Feels like `AppendCertsFromPEM` should be returning and error (which gets passed on here).